From 978f00c0a9f17de9cf17b205a741222084261d3a Mon Sep 17 00:00:00 2001
From: Ioana Ciornei <ioana.ciornei@nxp.com>
Date: Tue, 24 Oct 2017 16:29:17 +0000
Subject: [PATCH 01/12] scripts: move shebang on the first line

If the shebang is not placed on the first line of the shell script
it will be accounted only as a comment.

Signed-off-by: Ioana Ciornei <ioana.ciornei@nxp.com>
---
 scripts/ls-main | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/ls-main b/scripts/ls-main
index 588c736..09bef5a 100755
--- a/scripts/ls-main
+++ b/scripts/ls-main
@@ -1,3 +1,5 @@
+#!/bin/bash
+
 # Copyright 2013-2016 Freescale Semiconductor Inc.
 # Copyright 2017 NXP
 
@@ -30,8 +32,6 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-#!/bin/bash
-
 ##	Restool wrapper script
 #
 # Prerequisites:
-- 
2.14.1

